src/router/router.js

```diff
@@ -22,18 +22,24 @@
 
 import Vue from 'vue'
 import Router from 'vue-router'
-import { generateUrl } from '@nextcloud/router'
+import { getRootUrl, generateUrl } from '@nextcloud/router'
 import MainView from '../views/MainView.vue'
 import NotFoundView from '../views/NotFoundView.vue'
 import WelcomeView from '../views/WelcomeView.vue'
 
 Vue.use(Router)
 
+const webRootWithIndexPHP = getRootUrl() + '/index.php'
+const doesURLContainIndexPHP = window.location.pathname.startsWith(webRootWithIndexPHP)
+const base = generateUrl('/', {}, {
```

Comment:
shouldn't this contain '/apps/spreed' or at least '/s'?

Reply:
We use index.php/apps/spreed and index.php/call/...

So it needs to be empty